bundles/event_admin/event_admin/README.md

```diff
@@ -0,0 +1,107 @@
+limitations under the License.
+-->
+
+## Event Admin
+
+The Event Admin provides the pubsub mechanism for in-process communication. It has implemented two delivery mechanisms:
+- Synchronous delivery
+- Asynchronous delivery
+
+In addition, the Event Admin provides an adapter for the celix framework-level event. It maps the celix framework-level event to the
+event admin pubsub model, and events are delivered asynchronously.
+
+### Cmake options
+
+    BUILD_EVENT_ADMIN=OFF
+
+### Conan options
+
+    build_event_admin=false
+
+### Software Design
+
+#### Synchronous Delivery
+
+In the synchronous delivery, the event handler will be notified in the caller's thread. And the `sendEvent` method will return after all the event handlers have been called.
+The diagram of synchronous delivery is shown in the following figure:
+
+![sync_delivery_seq.png](diagrams/sync_delivery_seq.png)
+
+#### Asynchronous Delivery
+
+In the asynchronous delivery, the event handler will be notified in a separate event-delivery thread. And the `postEvent` method will return immediately.
+The diagram of asynchronous delivery is shown in the following figure:
+
+![sync_delivery_seq.png](diagrams/async_delivery_seq.png)
+
+In the above figure, if the event handler sets the "event.delivery" property to "async.ordered", each event handler can hold 
+at most one event-delivery thread at a time, so that events can be delivered in order. If the event handler sets the 
+"event.delivery" property to "async.unordered", the event handler can hold multiple event-delivery threads at the same 
+time, so that events can be delivered in parallel.
+
+
+#### Event Adapter
+
+In order to present a consistent view of all the events occurring in the system, the Event Admin provides an adapter for the celix framework-level event.
+The events are mapped as follows:
+
+- **Celix Framework Event**
+
+For celix framework Event, we map the celix framework condition id to the event admin event topic. The event admin event properties are none.
+The details mappings are shown in the following table:
+
+| Celix Framework Condition ID       | Event Admin Event topic                     |
+|------------------------------------|---------------------------------------------|
+| CELIX_CONDITION_ID_FRAMEWORK_READY | "org/osgi/framework/FrameworkEvent/STARTED" |
+| CELIX_CONDITION_ID_FRAMEWORK_ERROR | "org/osgi/framework/FrameworkEvent/ERROR"   |
+
+- **Celix Bundle Event**
+
+For celix bundle Event, we map the celix bundle event type to the event admin event topic. And The event admin event properties are as follows:
+- bundle.id
+- bundle.symbolicName
+- bundle.version
+
+The details mappings are shown in the following table:
+
+| Celix Bundle Event Type        | Event Admin Event topic                      |
+|--------------------------------|----------------------------------------------|
+| CELIX_BUNDLE_EVENT_INSTALLED   | "org/osgi/framework/BundleEvent/INSTALLED"   |
```

Comment:
Also mentioned for the service names, but I prefer that "org.osgi" or "osgi" is not used (inspired by, but not a implementation of). 
So I would prefer topics like "celix/framework/BundleEvent/INSTALLED